Anforderungsworkshop 2304.md

```diff
@@ -0,0 +1,54 @@
+# Anforderungsworkshop 23.04. 
+
+
+Zeit: 13.20 - 14.30 
+
+## Anwesenheit
+
+- Frau Hauptmann
+- Leo Lindhorst
+- Oliver von Seydlitz
+- Denis Keiling
+- Hung Nguyen
+- Fabian Krehnke
+
+## 1. Aufgabenstellung 
+
+**Entwicklung eines SW‐Systems zur Unterstützung der Planung von Präsentationen im Kontext des Moduls SE II.**
+
+Inhalt des Gesprächs war die Anforderungsanalyse mit dem Auftraggeber um die Ziele und Rahmenbedingungen des Projektes festzustellen.
+
+### 1.1 Ausgangslage
+
+Aktuell wird eine Excel-Tabelle für die Organisation genutzt. Diese liegt nur dem Auftraggeber vor. In dieser Tabelle sind die möglichen Zeiten für einen Termin festgehalten. Der Raum, in dem dieser stattfindet ist noch nicht bekannt. Termine und Termin Änderungen werden über E-Mail und Telefon in Absprache mit dem Auftraggeber ausgemacht und von ihm in die Liste eingetragen.
+
+### 1.2 Problem
+
+der Prozess bis zum festen Termin ist zeitaufwendig, da viel kommuniziert werden muss und Zeiträume durch die Prüfungsphase beschränkt sind.
+
+### 1.3 Projektziel
+
+Das Ziel ist eine auf den Universitätsserver hinterlegte Website (PHP). Diese Website dient als Organisationshilfe für Professoren und Studenten um Termine für Präsentationen zu vereinbaren.
+
+### 1.4 Akteure
+
+Prüfender, Prüfling(Gruppe).
+
+### 1.3 Optik und Nutzung
+
+Ein "Kalender" auf dem 15 Tage innerhalb der Prüfungsphase einsehbar sind. Die Tage sind mit Datum markiert und unterteilt in Doppelstunden gleich dem Stundenplan der HTW. Auf dieser Oberfläche können Termine mit 5 Zuständen eingetragen werden: *gesperrt*, *frei*, *reserviert*, *gebucht n.b.*, *gebucht b.*. Jeder Zustand besitzt eine eindeutige farbliche Markierung.

```

Comment:
Hier müsste erklärt werden, was *gebucht b.* bzw. *gebucht n.b.* ist. Ich bin mir nicht sicher, ob es in diesem Gespräch diese 5 Termine gab oder ob der 5. erst im Treffen mit Herrn Zirkelbach aufgetaucht ist